README.md

```diff
@@ -1,2 +1,81 @@
-# template-pkg
-A template repository for creating a Toit package
+# Package template
+
+A template repository for creating a Toit package.
+
+## Toit package
+Use `toit.pkg describe` or `toit pkg describe` (depending on which Toit
+variant you use) to see how https://pkg.toit.io extracts package
+information.
+
+Either add a `name: ...` entry to the package.yaml or change the title
+of this README to the package name.
```

Comment:
Suggestion: change the title (first line) of this README

Reply:
done.